### PR TITLE
feat(iFrame): Add a method for capturing screenshot of the large video

### PR DESCRIPTION
**Pull request by jallamsetty1**

### CODE REVIEW

File: react/features/large-video/actions.js [hunk 1]
```diff
@@ -20,6 +21,50 @@ import {
...
+
+       // wait for the video to load before drawing on to the canvas.
+       const promise = new Promise(resolve => {
+           element.onloadeddata = () => resolve();
+       });
+
+       return promise.then(() => {
+           ctx.drawImage(element, 0, 0, element.width, element.height);
+
+           return Promise.resolve(canvas.toDataURL('image/jpeg', 1.0));
```
Comment (saghul) [inline react/features/large-video/actions.js:63]: maybe destroy the video and canvas before returning?

Comment (jallamsetty1) [reply react/features/large-video/actions.js]: I added some cleanup code.

**Comment by jallamsetty1** [comment]
Thanks for the review @saghul, can you pls take a look again ?

File: react/features/large-video/actions.js [hunk 2]
```diff
@@ -20,12 +21,63 @@ import {
...
+        const { height, width } = track.getSettings() ?? track.getConstraints();
+        const canvasElement = document.createElement('canvas');
+        const ctx = canvasElement.getContext('2d');
+        const videoElement = document.createElement('video');
+
+        videoElement.height = parseInt(height, 10);
+        videoElement.width = parseInt(width, 10);
+        videoElement.autoplay = true;
+        videoElement.srcObject = videoStream;
+        canvasElement.height = parseInt(height, 10);
```
Comment (saghul) [inline react/features/large-video/actions.js:52]:
```suggestion
        canvasElement.height = videoElement.height
```

Comment (saghul) [reply react/features/large-video/actions.js:52]: How about this? (to avoid the parseInt twice)

**Review by saghul — APPROVED** [review]
LGTM, left a tiny comment, feel free to ignore it.

**Review by saghul — APPROVED** [review]

**Pull request merged**